tests/test_agent_manager.py

```diff
@@ -336,10 +369,27 @@ async def dummy_status():
             {"name": "agent1", "paused": True, "last_failover": "", "primary": "", "environment": env.id, "state": "paused"},
             {"name": "agent2", "paused": False, "last_failover": UNKWN, "primary": UNKWN, "environment": env.id, "state": "up"},
             {"name": "agent3", "paused": False, "last_failover": UNKWN, "primary": UNKWN, "environment": env.id, "state": "up"},
+            {"name": "agent1", "paused": True, "last_failover": "", "primary": "", "environment": env4.id, "state": "paused"},
+            {
+                "name": "agent2",
+                "paused": False,
+                "last_failover": UNKWN,
+                "primary": UNKWN,
+                "environment": env4.id,
+                "state": "up",
+            },
+            {
+                "name": "agent3",
+                "paused": False,
+                "last_failover": UNKWN,
+                "primary": UNKWN,
```

Comment:
`last_failover` and `primary` should be equal to an empty string right? `agent3` on environment `env4` never had a session.

Reply:
thx

Reply:
This one was not updated?

Reply:
push had failed, now it should be good